plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/guru/ReserveVmInterfaceVspAnswer.java

```diff
@@ -19,26 +19,26 @@
 
 package com.cloud.agent.api.guru;
 
-import java.util.List;
-import java.util.Map;
-
 import com.cloud.agent.api.Answer;
 import com.cloud.agent.api.Command;
 
 public class ReserveVmInterfaceVspAnswer extends Answer {
 
-    public List<Map<String, String>> _interfaceDetails;
-
-    public ReserveVmInterfaceVspAnswer(Command cmd, List<Map<String, String>> interfaceDetails, String details) {
+    public ReserveVmInterfaceVspAnswer(Command cmd, String details) {
```

Comment:
Why does this class exist?  It adds no state or behavior to the ``Answer`` base class.  Why not simply use ``Answer`` in this circumstance?